cocotb/triggers.py

```diff
@@ -669,11 +722,17 @@ class NullTrigger(Trigger):
     Primarily for internal scheduler use.
     """
 
-    def __init__(self, name=None, outcome=None):
+    def __init__(self, name=None, outcome=None, _outcome=None):
         super().__init__()
         self._callback = None
         self.name = name
-        self.__outcome = outcome
+        if outcome is not None:
+            warnings.warn(
+                "Passing the `outcome` argument and having that be the result of the ``await`` expression on this Trigger is deprecated.",
```

Comment:
```suggestion
                "Passing the `outcome` argument and having that be the result of the `await` expression on this Trigger is deprecated.",
```
We generally use light markdown syntax for output on the shell, right?